mirdata/rwc_popular.py

```diff
@@ -0,0 +1,374 @@
+    validate(dataset_path, data_home)
+    rwc_popular_data = {}
+    for key in track_ids():
+        rwc_popular_data[key] = Track(key, data_home=data_home)
+    return rwc_popular_data
+
+
+def _load_sections(sections_path):
+    if not os.path.exists(sections_path):
+        raise OSError('Could not find RWC Popular section annotations.')
+    begs = []
+    ends = []
+    secs = []
+
+    if os.path.exists(sections_path):
+        with open(sections_path, 'r') as fhandle:
+                reader = csv.reader(fhandle, delimiter='\t')
+                for line in reader:
+                    begs.append(float(line[0])/100)
+                    ends.append(float(line[1])/100)
+                    secs.append(line[2])
+        begs = np.array(begs)
+        ends = np.array(ends)
+        secs = np.array(secs)
+        data = utils.SectionData(begs,
+                                 ends,
+                                 secs)
+    else:
+        data = None
+
+    return data
+
+
+def _load_beats(beats_path):
+    if not os.path.exists(beats_path):
+        raise OSError('Could not find RWC Popular beat annotations.')
+    beat_times = []
+    beat_positions = []
+    mapping_positions = {'48': '1', '96': '2', '144': '3', '384': '4'}
+
+    if os.path.exists(beats_path):
+        with open(beats_path, 'r') as fhandle:
+            reader = csv.reader(fhandle, delimiter='\t')
+            for line in reader:
+                beat_times.append(float(line[0])/100)
+                beat_positions.append(mapping_positions[line[2]])
+        data = utils.BeatData(np.array(beat_times),
+                              np.array(beat_positions))
+    else:
+        data = None
+
+    return data
+
+
+def _load_chords(chords_path):
+    if not os.path.exists(chords_path):
+        raise OSError('Could not find RWC Popular chord annotations.')
+    begs = []
+    ends = []
+    chords = []
+
+    if os.path.exists(chords_path):
+        with open(chords_path, 'r') as fhandle:
+                reader = csv.reader(fhandle, delimiter='\t')
+                for line in reader:
+                    begs.append(float(line[0]))
+                    ends.append(float(line[1]))
+                    chords.append(line[2])
+        begs = np.array(begs)
+        ends = np.array(ends)
+        chords = np.array(chords)
+        data = utils.ChordData(begs,
+                               ends,
+                               chords)
+    else:
+        data = None
+
+    return data
+
+
+def _load_voca_inst(voca_inst_path):
+    if not os.path.exists(voca_inst_path):
+        raise OSError('Could not find RWC Popular vocal-instrument activity annotations.')
+    begs = []
+    ends = []
+    activity = []
+
+    if os.path.exists(voca_inst_path):
+        with open(voca_inst_path, 'r') as fhandle:
+            reader = csv.reader(fhandle, delimiter='\t')
+            raw_data = []
+            for line in reader:
+                if not line[0] == "Piece No.":
```

Comment:
`if line[0] != 'Piece No.'`

(side note - we switched to using single quotes in general)